gm4_lightning_rods/data/lightning_rods/functions/thrown.mcfunction

```diff
@@ -0,0 +1,9 @@
+# @s = lightning rod item on the ground
+
+execute if entity @s[tag=gm4_lightning_rod_boom] run function lightning_rods:boom
+
+data merge entity @s[tag=gm4_lightning_rod_count_1] {CustomName:"\"<1>\"",Tags:[gm4_lightning_rod_boom,gm4_lightning_rod_counting]}
```

Comment:
Do you think we should colour the name now that we can easily do this with the section sign? Maybe orange for 3 and 2 and then red for 1?

Reply:
Custom Names now support json, so we could color the names without needing the black magic of the section sign